frontend/online-shop/src/components/productDetails/ProductDetails.tsx

```diff
@@ -0,0 +1,54 @@
+import { useStyle } from "../../styles/productDetails/ProductDetailsStyle";
```

Comment:
You can add a `styles` extension to the name of this file instead of adding `Style` in its name. Something like `ProductDetails.styles.tsx`.